spec/yaml/swiftnav/sbp/signing.yaml

```diff
@@ -17,9 +17,20 @@ include:
   - gnss.yaml
 definitions:
   - MSG_ED25519_SIGNATURE:
-      id: 0x0C01
+      id: 0x0C03
       short_desc: ED25519 signature for groups of RTCM messages
       fields:
+        - stream_counter:
+            type: u8
+            desc: >
+              Signature message counter. Zero indexed and incremented with each signature message.
```

Comment:
Since SBP in general is supposed to be stateless, we should indicate that the starting value can be anything.  This will allow us to not worry about connection state (as much) in the implementation and clients can jump into something "mid stream" and work under assumption that first value isn't necessarily zero.

This would mean clients would have three states to implement:
- no counter seen -> record given value, assume stream valid
- counter seen and less than 255, compare with previous counter, if diff >1 then stream is invalid
- previous counter is 255, next counter should be 0 else stream is invalid